### ISSUE TITLE
Error message is overwritten by the generic form submission error message

### ISSUE THREAD

**Issue by ahmed-aurea** [crudlio/crudl-example-express#16]
Hi

As described in the documentations:
If any of the thrown errors contains an attribute message, this message will be displayed as a notification to the user.

But what happens is that the default error message "The form is not valid. Correct the errors and try again." is displayed instead.

Regards

**Comment by sehmaschine** [comment]
Yep, that's a typo. The error is shown as a field-level message. The notification is a general error message.
